pkg/synthetictests/events.go

```diff
@@ -72,19 +79,45 @@ func testDuplicatedEvents(events monitorapi.Intervals) []*ginkgo.JUnitTestCase {
 	const testName = "[sig-arch] events should not repeat pathologically"
 
 	displayToCount := map[string]int{}
+	ignoredCounts := map[string]int{}
 	for _, event := range events {
 		eventDisplayMessage, times := getTimesAnEventHappened(fmt.Sprintf("%s - %s", event.Locator, event.Message))
-		if times > 20 {
-			if knownEvents.MatchString(eventDisplayMessage) {
-				continue
-			}
+		if times == 0 {
```

Comment:
isn't times always at least 1 if we saw the event?  or is this a count of how many times it repeated after the first instance?

Reply:
That's an implementation oddity. This function returns `"", 0` if the message doesn't have the suffix `(X times)`. This suffix is added only if the k8s event met the condition `event.Count > 1`.